pkg/health/tcp_test.go

```diff
@@ -93,11 +95,13 @@ func TestTcpHealthChecker(t *testing.T) {
 	}
 	for _, test := range tests {
 		container := api.Container{
-			LivenessProbe: &api.LivenessProbe{
-				TCPSocket: test.probe,
+			LivenessProbe: &api.Probe{
+				Handler: api.Handler{
+					TCPSocket: test.probe,
+				},
 			},
 		}
-		params := container.LivenessProbe.TCPSocket
+		params := container.LivenessProbe.Handler.TCPSocket
```

Comment:
Can't `Handler.` be omitted, since it's an anonymous field?
https://golang.org/ref/spec#Struct_types
https://golang.org/ref/spec#Selectors